lib/pythonx.ex

```diff
@@ -290,6 +290,9 @@ defmodule Pythonx do
       for name <- defined do
         quote do
           unquote({String.to_atom(name), [], nil}) = Map.get(globals, unquote(name), nil)
```

Comment:
Or even:

    {String.to_atom(name), [generated: true], nil}

Reply:
Glorious, pushed to main, thank you!